presto-spi/src/main/java/com/facebook/presto/spi/type/CharType.java

```diff
@@ -31,6 +31,7 @@
         extends AbstractVariableWidthType
 {
     public static final int MAX_LENGTH = 65_536;
+    public static final CharType CHAR = new CharType(MAX_LENGTH);
```

Comment:
please create max length `CHAR` explicitly instead of adding field to SPI.

Reply:
Okay I've gotten rid of the change to SPI, and made a field in the test (because it's used in a couple of places).  Is this more what you were hoping for?